packages/integrations/cloudflare/src/server.advanced.ts

```diff
@@ -12,10 +12,10 @@ type Env = {
 	name: string;
 };
 
-export interface AdvancedRuntime {
+export interface AdvancedRuntime<T extends object> {
```

Comment:
Because it's a patch release, maybe a default generic argument should be added.
```suggestion
export interface AdvancedRuntime<T extends object = {}> {
```